handlers/bot_handler.py

```diff
@@ -24,8 +25,10 @@ class IntroCommand(Command):
     def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
         """Execute the Intro command."""
         try:
-            with open("intro_msg") as f:
-                message = f.read()
+            with open("./config.json") as f:
+                message = json.load(f)["intro_message"]
```

Comment:
can we use `json.load(f).get("intro_message", "")` so that we don't need to specify the config?

Reply:
I changed this - the bot now doesn't respond to the `!intro` command if there isn't a message specified in the config (or if `intro_msg` isn't defined in the config)